doc/tutorials/transformer/masked_modelling.ipynb

```diff
@@ -0,0 +1,277 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    # TODO: Move this to a separate configuration notebook. Here we just need to load this configuration from a store.
</pre><p>remove this line and either fix it or make a ticket instead</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/exasol/data-science-sandbox/pull/22/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
Please ignore this block in every notebook. Sorry, I should have explained this somewhere. This is a temporary piece of configuration code. It will be gone once the secret store is finished. We are working on that. Should be done very soon.